src/cromshell/list_outputs/command.py

```diff
@@ -0,0 +1,200 @@
+import logging
+
+import click
+import requests
+
+import cromshell.utilities.http_utils as http_utils
+import cromshell.utilities.io_utils as io_utils
+from cromshell.metadata import command as metadata_command
+from cromshell.utilities import command_setup_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="list-outputs")
+@click.argument("workflow_ids", required=True, nargs=-1)
+@click.option(
+    "-d",
+    "--detailed",
+    is_flag=True,
+    default=False,
+    help="Get the output for a workflow at the task level",
+)
+@click.option(
+    "-j",
+    "--json-summary",
+    is_flag=True,
+    default=False,
+    help="Print a json summary of outputs, including non-file types.",
+)
+@click.pass_obj
+def main(config, workflow_ids, detailed, json_summary):
+    """List all output files produced by a workflow."""
+
+    LOGGER.info("list-outputs")
+
+    return_code = 0
+
+    for workflow_id in workflow_ids:
+        command_setup_utils.resolve_workflow_id_and_server(
+            workflow_id=workflow_id, cromshell_config=config
+        )
+
+        if not detailed:
+            if json_summary:
+                io_utils.pretty_print_json(
+                    format_json=get_workflow_level_outputs(config).get("outputs")
+                )
+            else:
+                print_file_like_value_in_dict(
```

Comment:
Does this handle `Array[File]` outputs?

Reply:
It should handle `output_name: File` and `output_name: [File, File,..]`